Sources/BagbutikGenerator/Renderers/ObjectSchemaRenderer.swift

```diff
@@ -62,39 +62,37 @@ public class ObjectSchemaRenderer: Renderer {
                 structContent.append(renderInitializer(parameters: deprecatedPublicInitParameters, deprecated: true, content: { createInitContent(propertiesInfo.deprecatedPublicInitPropertyNames) }))
             }
             structContent.append(renderInitializer(parameters: propertiesInfo.publicInitParameters, content: { createInitContent(propertiesInfo.publicInitPropertyNames) }))
-            if propertiesInfo.needsCustomCoding {
```

Comment:
The following changes look more complicated than they are. This condition was removed causing the indentation of the following lines to change.

Reply:
This is not a problem, when I use GitHub's "Hide whitespace changes" ;)